packages/forge2d/lib/src/collision/shapes/circle_shape.dart

```diff
@@ -55,11 +55,10 @@ class CircleShape extends Shape {
     int childIndex,
     Vector2 normalOut,
   ) {
```

Comment:
Renamed some local variables to be consistent with the `testPoint` method above (easier to compare).